_posts/2020-07-13-our-code-review-etiquette.md

```diff
@@ -0,0 +1,79 @@
+- Ask for clarification: <i>"I didn't understand. Can you clarify?"</i>
+- Avoid selective ownership of code: "mine", "not mine", "yours"
+- Avoid using terms that could be seen as referring to personal traits: "dumb", "stupid", “not smart”. Assume everyone is intelligent and well-meaning.
+- Be explicit. Remember people don't always understand your intentions online.
+- Be humble: <i>I'm not sure - let's look it up.</i>
+- Don't use hyperbole: "always", "never", "endlessly", "nothing"
+- Don't use sarcasm. Even if it is your buddy.
+- Keep it real. If emoji, animated gifs, or humor aren't you, don't force them. If they are, use them with aplomb.
+- Talk synchronously (e.g. chat, screen sharing, in person) if there are too many comments like "I didn't understand" or "Alternative solution". Then post a follow-up comment summarizing the discussion.
+- Before submitting your review make sure you have considered <a href="https://www.ombulabs.com/blog/software-development/software-quality/our-definition-of-done.html" target="_blank">Our Definition of "Done"</a>.
+
+#### When you are reviewing
+
+- Understand why the change is necessary (fixes a bug, improves the user experience, refactors the existing code).
+- Be respectful. Keep in mind that there’s a person on the other side of the wire, not a machine, and that it’s hard to understand written words with little context. Avoid letting anger and frustration leak into the review, even if you feel it is justified. There’s no good outcome in those situations.
+- Communicate which ideas you feel strongly about and those you don't.
+- Explain why. Unless the change is about an extremely obvious mistake, explain why you’re suggesting it.
+- Identify ways to simplify the code while still solving the problem.
+- If discussions turn too philosophical or academic, move the discussion to another channel (Slack or email may be better?). In the meantime, let the author make the final decision about the implementation.
+- Avoid trivialities. Think to yourself: all things considered, does it actually matter? Is the cost of the author’s time, and the potential debate, really worth it?
+- Praise the good work. You can praise the logic, design, code organization, or whatever else that you honestly felt was well done.
+- Offer alternative implementations, but assume the author already considered them: <i>"What do you think about using a custom validator here?"</i>
+- When trying to offer alternative solutions, always link to resources if possible to help ensure that the claims being raised are validated.
+- Seek to understand the author's perspective.
+- Read the commit messages if you don't understand something. It's useful to try to understand why the author decided to do something.
+- It’s OK to say it’s all good. That doesn't hurt your reputation or mean that you are a less thoughtful reviewer. If the code is okay, just sign off on the pull request with a 👍 or "Ready to merge" comment.
```

Comment:
It would be nice to link to https://www.ombulabs.com/blog/agile/learning/pull-requests/submitting-prs.html

Maybe here:
```suggestion
- It’s OK to say it’s all good. That doesn't hurt your reputation or mean that you are a less thoughtful reviewer. If the code is okay, just sign off on the [pull request](https://www.ombulabs.com/blog/agile/learning/pull-requests/submitting-prs.html) with a 👍  or "Ready to merge" comment.
```